README.md

```diff
@@ -42,6 +42,7 @@ The following env variables need to be passed to the container:
 * `SMTP_PORT` (Optional, Default value: 587) Port address of the SMTP server to use.
 * `SMTP_USERNAME` Username to authenticate with.
 * `SMTP_PASSWORD` Password of the SMTP user.
+* (optional) `SMTP_PASSWORD_FILE` File Ref to load SMTP_PASSWORD, to aviod passwords in enviorment variables.
```

Comment:
Please fix typos and rephrase this to make it more clear that this new env var contains the path to a file that contains the smtp password instead of setting it on SMTP_PASSWORD variable.